server/mock/datastore.go

```diff
@@ -15,6 +15,11 @@ type Store struct {
 	DataStore
 }
 
+func (m *Store) EnrollOrbit(ctx context.Context, hardwareUUID string, orbitNodeKey string) (*fleet.Host, error) {
+	//TODO implement me
```

Comment:
Ask someone in #help-engineering for assistance in code-generating this.

Reply:
Delete this and then run `make generate-mock`